lib/checks/keyboard/landmark-is-top-level.json

```diff
@@ -0,0 +1,11 @@
+{
+  "id": "landmark-is-top-level",
+  "evaluate": "landmark-is-top-level.js",
+  "metadata": {
+    "impact": "moderate",
```

Comment:
By making this check more generic, is this impact level still accurate?

Reply:
Yes, this should be fine.